lib/mocha-hooks.js

```diff
@@ -86,15 +86,15 @@ afterEach( async function() {
 			} );
 		},
 		err => {
-			slackNotifier.warn( `Could not take screenshot due to error: '${ err }'`,
-				{ suppressDuplicateMessages: true }
-			);
+			slackNotifier.warn( `Could not take screenshot due to error: '${ err }'`, {
+				suppressDuplicateMessages: true,
```

Comment:
This needs to be in the brackets

Reply:
weird, I don't remember updating this but I'll make the change, thanks

Reply:
@bsessions85 it is in brackets, the linter just formatted it differently